data/docs/totem-releases-first-set-of-open-data/totem-releases-first-set-of-open-data.md

```diff
@@ -0,0 +1,24 @@
+Today, the TOTEM Collaboration published the first data set on the CERN Open
+Data Portal. Data recorded by TOTEM is used to measure the total cross section
+and to study elastic scattering in proton-proton collisions at the LHC for
+different centre-of-mass energies. The data set contains only the information
+from the Roman-Pot detectors. This data can be used to study elastic scattering
+at a centre-of-mass energy of 7 TeV. The open data release is accompanied by an
+example analysis showing how to measure the elastic scattering cross section as
+a function of the momentum transfer between the incoming protons and the total
+cross section.
+
+This release notably contains:
+
+- [2011 Proton scattering data from Roman Pots at energy sqrt(s) = 7 TeV](/record/84000)
+- [Measurement of proton-proton elastic scattering and total cross-section at sqrt(s) = 7 TeV](/record/84001)
+
+<p align="center">
+  <img src="/static/docs/totem-releases-first-set-of-open-data/totem-roman-pots-in-the-lhc-tunnel.jpeg/"
```

Comment:
Typo in the src? It ends with a '/'

Reply:
Yes, I'll fix it. Locally it works fine, so it went unnoticed...